library/src/main/java/com/datatorrent/lib/io/fs/LineByLineFileInputOperator.java

```diff
@@ -0,0 +1,55 @@
+package com.datatorrent.lib.io.fs;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import org.apache.hadoop.fs.Path;
+
+import com.datatorrent.api.DefaultOutputPort;
+
+/**
+ * This is an extension of the {@link AbstractFileInputOperator} that outputs the contents of a file line by line.&nbsp;
+ * Each line is emitted as a separate tuple in string format.
+ * <p>
+ * The directory path where to scan and read files from should be specified using the {@link #directory} property.
+ * </p>
+ * @displayName File Line Input
+ * @category Input
+ * @tags fs, file, line, lines, input operator
+ */
+public class LineByLineFileInputOperator extends AbstractFileInputOperator<String>
```

Comment:
Thought this might be more clearer.. FileLine words next to each other always felt a little weird.